src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -299,12 +330,40 @@ public override HttpClient GetObserverClient()
         /// </summary>
         private async Task<string> Get(string path)
         {
+            if (string.IsNullOrWhiteSpace(path))
+                throw new ArgumentNullException(nameof(path));
+
             var request = new HttpRequestMessage(HttpMethod.Get, $"https://support-bay-api.azurewebsites.net/observer/{path}?api-version=2.0");
             request.Headers.TryAddWithoutValidation("Authorization", await DataProviderContext.SupportBayApiObserverTokenService.GetAuthorizationTokenAsync());
             var response = await GetObserverClient().SendAsync(request);
             response.EnsureSuccessStatusCode();
             var result = await response.Content.ReadAsStringAsync();
             return result;
         }
+
+        public override Task<JObject> GetAdminSitesByHostNameAsync(string stampName, string[] hostNames)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override Task<JObject> GetAdminSitesBySiteNameAsync(string stampName, string siteName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override Task<string> GetStorageVolumeForSiteAsync(string stampName, string siteName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override Task<IEnumerable<object>> GetAppServiceEnvironmentDeploymentsAsync(string hostingEnvironmentName)
```

Comment:
Let's just get rid of these not implemented methods. I haven't heard any one needing them.